.travis.yml

```diff
@@ -2,6 +2,15 @@ language: node_js
 jobs:
   include:
 
+  - stage: check-pr
+    if: type = pull_request
+    node_js: lts/*
+    script:
+    - echo $TRAVIS_BRANCH
+    - echo $TRAVIS_PULL_REQUEST_BRANCH
+    - git diff --name-only HEAD...$TRAVIS_BRANCH
+    env: Performs sanity checks before building pull requests
```

Comment:
Ok, doesn't work. Idea is that the stage runs for PRs only and that a git command is used to get a list of all files changed between the respective AssemblyScript base branch and the PR, whatever branch it comes from.

Someone around familiar with these kind of things? :)